lib/main.dart

```diff
@@ -16,10 +16,10 @@ class MyApp extends StatelessWidget {
       debugShowCheckedModeBanner: false,
       title: 'Business Travel Tracker',
       theme: ThemeData(
-          backgroundColor: Color.fromARGB(255, 195, 231, 228),
```

Comment:
are you sure?

Reply:
Yes - this is the best solution, ofc - as for now.